site/standalone/installing.md

```diff
@@ -62,6 +78,11 @@ The Kubernetes deployment configuration file
 Flux daemon, but you'll need to edit it first, at least to supply your
 own configuration repo (the `--git-repo` argument).
 
+### Helm users
+
+Helm users need to provide also `--git-path` with path containing
+FluxHelmResource manifests.
```

Comment:
This seems bound to cause confusion: is it saying that people need to add another `--git-path` argument? Or that the flux daemon won't work unless you provide a different path that has `FluxHelmResource` manifests in it? Or something else.

Reply:
I think that should be a `FluxHelmRelease`, in any case.

Reply:
Starting with the last comment: I shall correct the typo.

The requirement until today was that the repo containing Charts will have charts under one path and FluxHelmRelease manifests under another. Your implementation today made this requirement redundant. Will remove the text.